### PR TITLE
Improve text and gramamr in the ApiTokenList component

### PR DESCRIPTION
**Pull request by juno-w**
Issue Number: #3444

## PR Details
Better text
```
API tokens are used to authenticate requests to the Amplication API. They are specifically required for Amplication CLI.
Tokens are valid for 30 days following creation or last use. The 30 day expiration period automatically refreshes with each API.
```

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Settings/ApiTokenList.tsx [hunk 1]
```diff
@@ -78,11 +78,9 @@ export const ApiTokenList = React.memo(() => {
         </Button>
       </div>
       <div className={`${CLASS_NAME}__message`}>
-        API tokens are used to authenticate requests to Amplication API,
-        specifically it is required to use Amplication CLI.
+        API tokens are used to authenticate requests to the Amplication API. They are specifically required for Amplication CLI.
```
Comment (arielweinberger) [inline packages/amplication-client/src/Settings/ApiTokenList.tsx:81]:
```suggestion
        API tokens are used by the Amplication CLI to make requests to the API.
```
I think in this case, less is more.

**Review by arielweinberger — CHANGES_REQUESTED** [review]

**Review by arielweinberger — APPROVED** [review]

**Comment by yuval-hazaz** [comment]
Thank you for your first contribution with Amplication 🚀

**Comment by souravjain540** [comment]
your name and email @juno-w ?

**Comment by juno-w** [comment]
> your name and email @juno-w ?

Dm'ed your Twitter :)

**Pull request merged**